### ISSUE TITLE
Stage clearing when player leaves - no content remains - unable to delete

### ISSUE THREAD

**Issue by vickis** [upstage-org/upstage#1021]
Heya - I am having an issue with the stage set up not remaining for visitors after any players have left
I have tried duplicating and then making a new stage (also as below I now cannot delete the other stages....
please could the Refinery instance be given a once over to check it is working correctly? 
 
<img width="1217" alt="image" src="https://user-images.githubusercontent.com/4249450/194172129-62cc2a1f-9585-464f-94d4-ec616219ced4.png">

We hope to do the live cross when the firing opens (tipped for sunday ) but with this current snowfall it may occur sooner - saturday perhaps

**Comment by vickis** [comment]
i have been out for a project and came back to have another look at what might be happening - what I notice is that there is not player showing in (even though I am in the stage below are two comparisons 
<img width="1280" alt="image" src="https://user-images.githubusercontent.com/4249450/194195818-c658a0bc-fa9b-4d90-9e01-957fc35bed2c.png"> (here I am logged in and have arranged the stage and saved as a scene - below i have entered as a guest to see what it looks like ... but it shows no player and the stage is blank 

<img width="1256" alt="image" src="https://user-images.githubusercontent.com/4249450/194195978-978c797a-f7dd-4725-8799-85aefb64c741.png">

**Comment by vickis** [comment]
I have tried logging out and in again 
<img width="1270" alt="image" src="https://user-images.githubusercontent.com/4249450/194196362-afb2c531-0382-449b-904e-8c01337aaa78.png">

<img width="1251" alt="image" src="https://user-images.githubusercontent.com/4249450/194196613-b54e2748-9cb2-4129-9e7c-3b9fcc40bf33.png">

<img width="1277" alt="image" src="https://user-images.githubusercontent.com/4249450/194196694-f405d8b9-a36e-4062-a3d0-9a98c90fcaf8.png">

**Comment by helenvarley** [comment]
i just went in to https://refinery.upstage.live/demo and put some media on the stage, then went out and came back and found it was blank. definitely something not right! (this was in both firefox & chromium on linux).
i checked upstage.live, meta & dev-app, on all of these the stages are loading with whatever media has been left on them, so this bug is unique to refinery.upstage.live

**Comment by vickis** [comment]
helen and i are just testing this is what i see 
<img width="1272" alt="image" src="https://user-images.githubusercontent.com/4249450/194819854-37ff603b-c25d-40e1-84f6-dd54f5ce71c3.png">

helen sees something different

**Comment by helenvarley** [comment]
vicki & i are testing at the same time. the bug is still happening. we are both in there, but i am only seeing myself in the player/audience counter. i can see vicki's chat, but i'm not seeing anything until she does something after i've entered. AND the stage status is showing as offline.
![Screenshot from 2022-10-10 09:48:47](https://user-images.githubusercontent.com/3437324/194820195-41e1b74b-e6cd-4d99-b559-8379c76f09fb.png)
![Uploading Screenshot from 2022-10-10 09:52:37.png…]()

**Comment by vickis** [comment]
i noted in slack that the server was done briefly because Janja's card (refinery) declined so i had to add mine - everything came back except this issue was somehow introduced

**Comment by vickis** [comment]
Stages are not working properly - if I am logged in and set up a stage when i  open another browser and enter as audience in  there is nothing there 
once there has been a change it appears for audience  - but if either audience or player leave the stage on return it is blank again 
no media remains

**Comment by helenvarley** [comment]
note: **this bug is still specific to refinery.upstage.live** - i've just checked upstage.live and meta.upstage.live, and on both the media is all there.

**Comment by gloriajw** [comment]
Upstage and refinery are running the same version. This is really odd.

**Comment by helenvarley** [comment]
yes it is. as vicki said, there was a period when the refinery's DO payment didn't go through & the instance went offline. it seems that this problem has appeared since the payments were restarted & the instance came back online. could that have anything to do with it? maybe some service has not restarted?

**Comment by vickis** [comment]
Yes - the stage on first entering shows as offline (perhaps similar to what
is/was happening on Meta - but then comes online after a second or two
I am also unable to delete stages as above get the error message ...

**Comment by vickis** [comment]
I have tested in UpStage.live and it is working as expected :D so the issue is the refinery instance only asaik

**Comment by vickis** [comment]
https://upstage.live/watery
